### PR TITLE
vipsister23.com

### LINKED ISSUE #121347: vipsister23.com
### Issue URL (Annoyance)
 [http://vipsister23.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fvipsister23.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>
<details>
  <summary>Screenshot 3</summary>

  Screenshot 3
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard/AdGuard Pro for iOS v4.4.0
Browser: | Safari
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/>AdGuard Social Media<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/>AdGuard Japanese

### PR DESCRIPTION
**Pull request by SKEIDs**
Fix #121347

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -130,7 +129,7 @@ enfant.living.jp##.ad-inread
 arine.jp##.Entry-top-ad
 news.yahoo.co.jp##a[data-cl-params$="cl_link:bnr_b;"]
 m.malaysiakini.com##.items-center[style="height:300px"]
-kabumatome.doorblog.jp##.plugin-free_area
+vipsister23.com,kabumatome.doorblog.jp##.plugin-free_area
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:132]: Hm, apparently this is not ad leftover but pls check @zloyden

Comment (zloyden) [reply MobileFilter/sections/specific_web.txt:132]: Looks like this is not a leftover - I do not see ads without AdGuard and with Japanese IP

Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt]: ok, i removed it

File: SocialFilter/sections/specific.txt [hunk 2]
```diff
@@ -2004,7 +2004,8 @@ indosport.com##div[title^="Bagikan Artikel Ini"]
 otvet.mail.ru##a[title="Поделиться"]
 houzz.ru##.hz-editorial-gallery-action-bar__share
 blog.esuteru.com##.comment-v2 + aside.section-box
-blog.esuteru.com##.plugin-twitter_follow_button
+vipsister23.com,blog.esuteru.com##.plugin-twitter_follow_button
```
Comment (Yuki2718) [inline SocialFilter/sections/specific.txt:2007]: This also is empty even if AG is turned off, tho I think it's acceptable having the rule.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
Add `||ad-spire.net^$third-party` to Japanese adserver and `vipsister23.com###lite_link` to Mobile filter.

File: SocialFilter/sections/specific.txt (hunk 2 above)
Comment (SKEIDs) [reply SocialFilter/sections/specific.txt:2007]: I see

File: MobileFilter/sections/specific_web.txt (hunk 1 above)
Comment (zloyden) [reply MobileFilter/sections/specific_web.txt]: @SKEIDs in my view, we can leave this rule, please revert it

File: MobileFilter/sections/specific_web.txt [hunk 3]
```diff
@@ -906,8 +905,9 @@ gadget2ch.com###main > .posts-inner:first-child
 gadget2ch.com###page > center
 abendzeitung-muenchen.de##.artikel-detail-mobile-ad
 fpsjp.net,ryukyushimpo.jp###sp_top_header
-tsubamesoku.blog.jp,blog.esuteru.com###article-1_5
+sutekinakijo.com,news.2chblog.jp,kyousoku.net,waranote.livedoor.biz,vipsister23.com,tsubamesoku.blog.jp,blog.esuteru.com###article-1_5
 segodnya.ua##.branding-under-menu
+vipsister23.com###lite_link
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:910]: @SKEIDs Existing: `news4wide.net,chomagireko.blog.jp###lite_link`

Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt]: sorry, fixed

**Review by Yuki2718 — COMMENTED** [review]
I noticed there are three `#article-1_5` rules in Mobile filter:
```
sutekinakijo.com,news.2chblog.jp,kyousoku.net,waranote.livedoor.biz###article-1_5
tsubamesoku.blog.jp,blog.esuteru.com###article-1_5
girlsvip-matome.com,onecall2ch.com,gahalog.2chblog.jp,michaelsan.livedoor.biz,chaos2ch.com,bipblog.com,news4vip.livedoor.biz,akb48matomemory.com,doorblog.jp,jin115.com,himasoku.com,blog.livedoor.jp##.article-list-outer > li#article-1_5
```
I checked quickly and apprently all these can be merged into one rule of `...###article-1_5`; i.e. `article-list-outer > li` specification is not needed. Furthermore, it seems `#article-1_5` is always leftover and there's no `#article-1_4` or `#article-2_5` so I think `sutekinakijo.com,news.2chblog.jp,`, `gahalog.2chblog.jp`, `waranote.livedoor.biz`, `news4vip.livedoor.biz`,  and `tsubamesoku.blog.jp` can be trimmed down to `2chblog.jp,livedoor.biz,blog.jp###article-1_5` so that all subdomain will be covered, particularly as we already did so for `blog.livedoor.jp` and `doorblog.jp`.

**Comment by SKEIDs** [comment]
> I noticed there are three #article-1_5 rules in Mobile filter:

I see. I am going to merge those rules.

File: MobileFilter/sections/specific_web.txt [hunk 4]
```diff
@@ -1624,7 +1622,7 @@ matometanews.com###category_ranking
 newsoku.blog##div[style="width:100%;height:500px; overflow: hidden;"]
 newsoku.blog##footer > div.widget_mobile_ad ~ div[class^="widget"]
 news4vip.livedoor.biz###f984a
-girlsvip-matome.com,onecall2ch.com,gahalog.2chblog.jp,michaelsan.livedoor.biz,chaos2ch.com,bipblog.com,news4vip.livedoor.biz,akb48matomemory.com,doorblog.jp,jin115.com,himasoku.com,blog.livedoor.jp##.article-list-outer > li#article-1_5
+girlsvip-matome.com,sutekinakijo.com,tsubamesoku.blog.jp,onecall2ch.com,jin115.com,chaos2ch.com,bipblog.com,akb48matomemory.com,himasoku.com,blog.esuteru.com,kyousoku.net,vipsister23.com,2chblog.jp,doorblog.jp,livedoor.biz,blog.livedoor.jp###article-1_5
```
Comment (Yuki2718) [inline MobileFilter/sections/specific_web.txt:1625]: `tsubamesoku.blog.jp` => `blog.jp`

Comment (SKEIDs) [reply MobileFilter/sections/specific_web.txt:1625]: sorry, fixed

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**